assets/icons/grid-view-icon.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * WordPress dependencies
+ */
+import { Path, SVG } from '@wordpress/components';
+
+export const GridViewIcon = () => (
```

Comment:
The toolbar icons are also pretty broken, because it's wrapped in a function but not used as a component (`<GridViewIcon />`). But this should be fixed by switching to the core icon library (which just exports the SVG)

<img width="569" alt="image" src="https://user-images.githubusercontent.com/176949/112495473-350a5080-8d84-11eb-8b05-1ea01993edda.png">

Reply:
Same fix above.